docs: tags filter replaces profiles & update screenshots (#5334)
spectrocloud/librarium: docs/docs-content/clusters/cluster-management/cluster-map-filters.md

5. Click on **Clear All**. The filters are removed and the cluster list displays all clusters in your selected scope.

> 6. Enter a cluster tag in the **Tags** field. The cluster list displays all clusters deployed with the specified cluster
>    tag.

Author: ```suggestion
6. Enter cluster tags in the **Tags** field. The cluster list displays all clusters deployed with the specified cluster tags.
```

>    tags.

Author: ```suggestion
   tag.
```

6. Click on **Clear All**. The filters are removed and the map displays all clusters.

> 7. Enter a cluster tag in the **Tags** field. The map displays all clusters deployed with the specified cluster tag.

Author: ```suggestion
7. Select cluster tags in the **Tags** field. The map displays all clusters deployed with the specified cluster tags.
```
